docs/2_2_common_schema.adoc

```diff
@@ -672,16 +688,29 @@ The algebraic relationship to the inputs is defined via the `dependencies` attri
 - `"local"`: Local variable that is calculated from other variables or is a continuous-time state (see <<ModelStructure>>).
 It is not allowed to use the variable value in another model or slave.
 
-- `"independent""`: The independent variable (usually `time`).
+- `"independent"`: The independent variable (usually `time`).
```

Comment:
I suggest to only use `independent` (w/o double quotes) except you cite a complete XML tag.

Reply:
I just fixed the typo here (two closing " instead of one "). This is part of a list of all allowed attribute values, where each value is written in "value".
I suggest that this pull request just contains the missing changes of the doc file.
This is a more general style issue.